hw3/02-charts.js

```diff
@@ -28,18 +28,44 @@ const borderColors = [
 
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const section = document.querySelector('section');
 
-const renderChart = () => {
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    const housesCounts = {};
+    const individualVals = [];
+    const houses = [];
+    const counts = [];
```

Comment:
Maybe these need to be let instead of const? Aren't you adding to them as you go through the characters, so they would be changed?